### PR TITLE
[ES] Miscellaneous orthographic fixes

### PR DESCRIPTION
**Pull request by anticzunzun**
Several orthography-related changes to the Spanish translation.

### CODE REVIEW

File: locales/es.yml [hunk 1]
```diff
@@ -2,18 +2,18 @@
 es:
   2fa:
     5strike:
-      return: Volver a la portada
+      return: volver a la portada
```
Comment (rakvat) [inline locales/es.yml]: Why are you changing the capitalization? This changes the style.

Comment (rakvat) [reply locales/es.yml]: @anticzunzun ok cool. Thank you for checking the English version.

File: locales/es.yml [hunk 2]
```diff
@@ -2,18 +2,18 @@
...
       subtitle: Caramba, por razones de seguridad, tendrá que esperar dos horas antes de poder intentar conectarse de nuevo.
       title: Regla de los cinco intentos
     cta: Soy yo, ¡déjame entrar!
     error: "<strong>Operador Kinky:</strong> Doh&hellip; el código de verificación de arriba parece incorrecto. Por favor, compruebe que el número anterior coincide con lo que recibió de nosotros y vuelva a intentarlo."
     label: Código de autentificación
     page_title: Autentificación en dos pasos
     subtitle: "¡2-Factorízame cariño!"
-    title: Código Secreto
+    title: Código secreto
```
Comment (rakvat) [inline locales/es.yml]: in this case the English version is "Secret Code". We use capitalization as style in titles. Why did you change it here?

Comment (anticzunzun) [reply locales/es.yml]: That corresponds to commit cac266f24c2db354166189f99d3d05b58d6a2ba0 in which Spanish-style title/work/UI-element conventions are enforced. My decision was based on two points: there were more than a few occurrences where this was not being respected; and other language files seem to have adopted their regional style (e.g. French) so, if searching for consistency, leaning to this side seemed the better choice.

Comment (rakvat) [reply locales/es.yml]: ok, thank you!
